project-settings/teams-and-users/manage-roles.md

```diff
@@ -86,6 +86,25 @@ _Collection permissions_ allow you to enable/disable the following collection-sp
 - _Delete_: delete a record of this collection
 - _Export_: export the list of records of this collection
 
+### Default permissions <a href="#default-permissions" id="default-permissions"></a>
+
+_Default permissions_ allow you to choose default permissions for each newly created Collection or Smart Action.
+
+
+Here is the initial configuration for a new role:
+* Smart Actions
+  |Trigger| Require approval  | Approve| Self approve| 
+  | :-: | :-: | :-: | :-: | 
+  | :heavy_check_mark: | - | - | - | - |
+* Collections:
+  | Read (list) | Read (details) | Create | Update | Delete | Export |
+  |  :-: |  :-: |  :-: |  :-: |  :-: |  :-: |
+  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | 
+
+{% hint style="info" %}
+For projects with high security standards, although it implies more manual opt-in activation, it is advised to reduce these default permissions at least in production, to prevent excessive rights in operators hands.
```

Comment:
Just a rewording for a clearer explanation:
```suggestion
For projects with high security standards, although this involves more frequent manual activation, it is advisable to reduce these default permissions, at least for your production environment. Il will avoid excessive accesses being automatically granted to operators, when a new Collection or Smart Action is released.
```
(I didn’t expect my comment words to be taken as is, sorry)

Reply:
Sorry I thought it looked great so I took it 😅